wp-content/themes/goonj-crm/functions.php

```diff
@@ -372,11 +372,35 @@ function goonj_handle_user_identification_form() {
 }
 
 function goonj_is_volunteer_inducted( $volunteer ) {
+	$optionValue = \Civi\Api4\OptionValue::get(TRUE)
+	->addWhere('option_group_id:name', '=', 'activity_type')
+	->addWhere('label', '=', 'Induction')
+	->execute();
+
+	$activityTypeId = $optionValue->first()['value'];
+
+	$completeStatusOptionValue = \Civi\Api4\OptionValue::get(TRUE)
+	->addWhere('option_group_id:name', '=', 'activity_status')
+	->addWhere('label', '=', 'completed')
+	->setLimit(25)
+	->execute();
+
+	$completeStatusId = $completeStatusOptionValue->first()['value'];
+
+	$unknownStatusOptionValue = \Civi\Api4\OptionValue::get(TRUE)
+	->addWhere('option_group_id:name', '=', 'activity_status')
+	->addWhere('label', '=', 'Unknown')
+	->setLimit(25)
+	->execute();
+
+	$unknownStatusId = $unknownStatusOptionValue->first()['value'];
```

Comment:
Let's remove this and directly use the `label` in the query. Because if you see we are making query on the basis of `label`, so if the `label` changes the query will not return the correct result. So why not use the label directly and prevent two API calls?